icon-registry/icons-static/action-stop-circle_x16.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path
+    fill-rule="evenodd"
+    clip-rule="evenodd"
+    d="M8 16C12.4183 16 16 12.4183 16 8C16 3.58172 12.4183 0 8 0C3.58172 0 0 3.58172 0 8C0 12.4183 3.58172 16 8 16ZM5 4C4.44772 4 4 4.44772 4 5V11C4 11.5523 4.44772 12 5 12H11C11.5523 12 12 11.5523 12 11V5C12 4.44772 11.5523 4 11 4H5Z"
+    fill="currentColor"
+    class="icon-light"
```

Comment:
Seems to me that the class should be "icon-dark", not "icon-light". @mabela416 

Does this even matter when the icon is using a single color? @ryanjwilke